Overlays/Windows/Libraries/hashing/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+append_to_overlay_var(WN_HASHING_XXHASH_CXX_FLAGS /wd4255)
+append_to_overlay_var(WN_HASHING_XXHASH_CXX_FLAGS /wd4668)
+append_to_overlay_var(WN_HASHING_XXHASH_CXX_FLAGS /wd4820)
+
+if(MSVC_VERSION GREATER_EQUAL 1920)
+    append_to_overlay_var(WN_HASHING_XXHASH_CXX_FLAGS /Qspectre)
```

Comment:
I'm wondering if I should just suppress this instead of enabling the compiler to "fix" spectre.